scripts/firestore_delete/main.py

```diff
@@ -0,0 +1,35 @@
+#The given script creates a CLoud Function to that is triggered when an image file is deleted from the street2sat-uploaded bucket
+#and then proceeds to delete the corresponding record from the Firestore DB.
```

Comment:
nit: Inline comments should start with space: https://peps.python.org/pep-0008/#inline-comments